common/api/components-react.api.md

```diff
@@ -3249,6 +3367,32 @@ export function useVirtualizedPropertyGridLayoutStorage<T extends Element>(): {
     restore: () => void;
 };
 
+// @beta
+export type Value = NumericValue | InstanceKeyValue | TextValue | BooleanValue | DateValue | EnumValue;
```

Comment:
Can we add new value types later if needed? I.e. for string literal type extensibility we do this: https://github.com/iTwin/appui/blob/89d6bf22b8110d74d01b55f6cbfc497559c8bcf1/ui/appui-react/src/appui-react/theme/ThemeId.ts#L38